lab1.js

```diff
@@ -67,24 +68,20 @@ assert(1 === 2, 'this is an example of a failing assertion. 1 does not equal 2.'
 */
 
 var sentence1 = 'More food please.';
-var sentence2 = 'Come over here so you can scratch my belly.';
-/*
- Your goal is to replace the words in the above sentences with 'chirp' The
- assertions at the end of this section should pass when you're done.
- HINT: the 'split' method on String will be useful.
-*/
-
-// TODO: part #1: use a for loop to replace the words in sentence 1 with
-// 'chirp' (10 points)
-
-// your code goes here
-
-// TODO: part #2: use a while or do-while loop to replace the words in sentence 2
-// with 'chirp' (10 points)
+var splitString = sentence1.split('');
+for (var i = 0; i < splitString.length; i++) {
+  splitString[i] = 'chirp';
+}
+sentence1 = splitString.join(' ');
 
-// your code goes here
+var sentence2 = 'Come over here so you can scratch my belly.';
+var splitString2 = sentence2.split(' ');
+var j = splitString2.length;
+while (j--) {
+  splitString2[j] = 'chirp';
+}
+sentence2 = splitString2.join(' ');
```

Comment:
check out the difference between 
'chirp chirp chirp'
and
'chirp chirp chirp.' //this is the one in the assert
Fix that, and I'm pretty sure your asserts will pass.